### ISSUE TITLE
File Handling not Supported

### ISSUE THREAD

**Issue by jwbear** [anthonyhtyao/varnaapi#1]
The structure command doesn't handle file input/ouput such as bpseq etc. Either a processing module could be added to preprocess bpseq or add handling for file i/o in Structure.

**Comment by jwbear** [comment]
Varna jar file not downloaded with pip install. The pip install looks for the jar in the install directory, but doesn't download or install it. It could probably be downloaded and set as default path as not doing this manually causes file not found errors.

**Comment by anthonyhtyao** [comment]
> Varna jar file not downloaded with pip install. The pip install looks for the jar in the install directory, but doesn't download or install it. It could probably be downloaded and set as default path as not doing this manually causes file not found errors.

Hi, this is made by design. It is also not suggested to download from external url during pip install. But we can think of including the jar in varnaapi source code.

**Comment by anthonyhtyao** [comment]
> The structure command doesn't handle file input/ouput such as bpseq etc. Either a processing module could be added to preprocess bpseq or add handling for file i/o in Structure.

I will include this in next release. For now, we can create a drawing class as follow to handle file input

```python
import varnaapi

# Disable auto check as structure length is unknown (unless parse the inputfile)
varnaapi.enable_hack()

class FileDraw(varnaapi.BasicDraw):
    def __init__(self, inputfile):
        super().__init__()
        self.inputfile = inputfile

    def _gen_input_cmd(self):
        return ['-i', self.inputfile]
```

The class `FileDraw` supports other member functions as `Structure` does

**Comment by anthonyhtyao** [comment]
Included in v1.2.0
